idseq_pipeline/commands/common.py

```diff
@@ -281,57 +280,38 @@ def wrapped_operation(*args, **kwargs):
     return wrapped_operation
 
 
-def execute_command_with_output(command, progress_file=None, timeout=None, grace_period=None):
+def execute_command(command, progress_file=None, timeout=None, grace_period=None, with_output=False):
```

Comment:
Previously we only supported timeout enforcement for execute_command_with_output.   Now we also support it for execute_command.